src/Utilities/WellKnownTypes.cs

```diff
@@ -151,6 +151,49 @@ internal static class WellKnownTypes
+        public const string SystemWebUIWebControlsBoundField = "System.Web.UI.WebControls.BoundField";
+        public const string SystemWebUIWebControlsCommandField = "System.Web.UI.WebControls.CommandField";
+        public const string SystemWebUIWebControlsDataControlField = "System.Web.UI.WebControls.DataControlField";
+        public const string SystemWebUIWebControlsDataGrid = "System.Web.UI.WebControls.DataGrid";
+        public const string SystemWebUIWebControlsDataGridColumn = "System.Web.UI.WebControls.DataGridColumn";
+        public const string SystemWebUIWebControlsHotSpot = "System.Web.UI.WebControls.HotSpot";
+        public const string SystemWebUIWebControlsHtmlForm = "System.Web.UI.WebControls.HtmlForm";
+        public const string SystemWebUIWebControlsImage = "System.Web.UI.WebControls.Image";
+        public const string SystemWebUIWebControlsImageField = "System.Web.UI.WebControls.ImageField";
+        public const string SystemWebUIWebControlsLoginStatus = "System.Web.UI.WebControls.LoginStatus";
+        public const string SystemWebUIWebControlsPagerSettings = "System.Web.UI.WebControls.PagerSettings";
+        public const string SystemWebUIWebControlsPanel = "System.Web.UI.WebControls.Panel";
+        public const string SystemWebUIWebControlsPanelStyle = "System.Web.UI.WebControls.PanelStyle";
+        public const string SystemWebUIWebControlsRadioButton = "System.Web.UI.WebControls.RadioButton";
+        public const string SystemWebUIWebControlsSiteMapDataSource = "System.Web.UI.WebControls.SiteMapDataSource";
+        public const string SystemWebUIWebControlsTableStyle = "System.Web.UI.WebControls.TableStyle";
+        public const string SystemWebUIWebControlsTreeNodeStyle = "System.Web.UI.WebControls.TreeNodeStyle";
+        public const string SystemWebUIWebControlsWebControl = "System.Web.UI.WebControls.WebControl";
+        public const string SystemWebUIWebControlsWebPartsDeclarativeCatalogPart = "System.Web.UI.WebControls.WebParts.DeclarativeCatalogPart";
+        public const string SystemWebUIWebControlsWebPartsGenericWebPart = "System.Web.UI.WebControls.WebParts.GenericWebPart";
+        public const string SystemWebUIWebControlsWebPartsPageCatalogPart = "System.Web.UI.WebControls.WebParts.PageCatalogPart";
+        public const string SystemWebUIWebControlsWebPartsWebPart = "System.Web.UI.WebControls.WebParts.WebPart";
+        public const string SystemWebUIWebControlsWebPartsWebPartZoneBase = "System.Web.UI.WebControls.WebParts.WebPartZoneBase";
+        public const string SystemWebUIWebControlsWebPartsWebZone = "System.Web.UI.WebControls.WebParts.WebZone";
+        public const string SystemWebUIWebControlsWebPartsZoneLinkButton = "System.Web.UI.WebControls.WebParts.ZoneLinkButton";
+        public const string SystemWebUIWebControlsWizard = "System.Web.UI.WebControls.Wizard";
+        public const string SystemWebUtilHttpEncoder = "System.Web.Util.HttpEncoder";
```

Comment:
Probably worth separating out all the fully qualified type names into a separate type, say `WellKnownTypeNames`, OR renaming all the methods below to be prefixed with `Get`. Does not need to happen as part of this PR though.